src/main/resources/static/css/styles.css

```diff
@@ -0,0 +1,90 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat:200,300,400,500,600,700');
+body, div, section, article, h2, header, footer{
+  box-sizing: border-box;
+  margin: 0px;
+}
+body{
+  font-family: "Montserrat", Helvetica, Arial, sans-serif;
+  font-size: 16px;
+  letter-spacing: 1.5px;
+  margin:0 auto;
+  max-width: 1337px;
+}
+header{
+  margin-top:24px;
+  padding: 24px;
+  text-align:center;
+}
+header h1{
+  font-size: 36px;
+  font-weight: 200;
+  color: rgba(0,0,0,.7);
+  margin: 0px 0px 0px 0px;
+  }
+header h3{
+  font-size: 16px;
+  font-weight: 400;
+  color: rgba(0,0,0,.2);
+  margin: 16px 0px 0px 0px;
+}
+footer{
+  clear:both;
+  padding: 24px;
+}
+footer h4{
+  font-size: 12px;
+  color: rgba(0,0,0,.4);
+}
+section{
+  width:100%;  
+  padding: 12px;
+}
+article{
+  color: white;
+  display: -webkit-box;
+  display: -moz-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
+  float: left;
+  width: 50%;
+  padding: 12px;
+}
+div{
+  position:relative;
+  min-height: 280px;
+  width: 100%;
+  background-color: background-color: #A9C9FF;
```

Comment:
Hey Simon! 

I think the only other thing to fix on this is to remove the repeated background-color: on line 57. then we should be in good shape to merge :)

Reply:
You'll need to do a 'git push' so it disappears from the pr

Reply:
I did a git push. I got an error message saying everything up-to-date